setup/owncloud.sh

```diff
@@ -32,28 +28,30 @@ if [ ! -f $STORAGE_ROOT/owncloud/config.php ] \
 	ln -sf $STORAGE_ROOT/owncloud/config.php /usr/local/lib/owncloud/config/config.php
 fi
 
-# Check if ownCloud dir exist, and check if version matches owncloud_ver (if either doesn't - install/upgrade)
-if [ ! -d /usr/local/lib/owncloud/ ] \
-	|| ! grep -q $owncloud_ver /usr/local/lib/owncloud/version.php; then
+InstallOwncloud() {
+        version=$1
+        hash=$2
```

Comment:
Would be good to add something like `Upgrading to ownCloud $version...` here, so that when there are multiple upgrades happening the console output is clear about what's happening when.